cli/features/steps/steps.ls

```diff
@@ -139,7 +139,9 @@ module.exports = ->
       | 'exo create application' => 'We are about to create a new Exosphere application'
       | 'exo create service'     => 'We are about to create a new Exosphere service'
       | 'exo add'                => 'We are about to add a new Exosphere service to the application'
-    @process.wait expected-text, done
+    @process.wait expected-text, ~>
+      DockerHelper.remove-container \exocom if expected-text is \exo-run
```

Comment:
Please add a comment describing why you need to wait here when `exo-run` is running